maze-game/TimeFilter.java

```diff
@@ -0,0 +1,39 @@
+import greenfoot.*;  // (World, Actor, GreenfootImage, Greenfoot and MouseInfo)
+
+/**
+ * Write a description of class TimeFilter here.
+ * 
+ * @author (your name) 
+ * @version (a version number or a date)
+ */
+public class TimeFilter extends Actor
+{
+    private int timer;
+    public TimeFilter()   //constructor to resize intro text to be a better fit for the maze
+    {
+        GreenfootImage intro = getImage();
+        intro.scale(1150, 950);
+        intro.setTransparency(80);
+    }
+    /**
+     * Act - do whatever the TimeFilter wants to do. This method is called whenever
+     * the 'Act' or 'Run' button gets pressed in the environment.
+     */
+    public void act()
+    {
+        timer ++;
+        if(timer<=500)
```

Comment:
When i tested, this seemed like maybe a little too long. thoughts?

Reply:
could shorten it if you want